contract-map.json

```diff
@@ -1,4 +1,32 @@
 {
+  "0xc20C9C13E853fc64d054b73fF21d3636B2d97eaB": {
```

Comment:
On which chain is deployed this contract?

Reply:
> On which chain is deployed this contract?

As referenced in the PR description, this is on Gnosis Chain.

Reply:
Can you separate contract on ETH from Gnosis to two separate PR?

Reply:
@MRabenda Yes, removed Gnosis Chain for this PR.